site/app/templates/notebook/Notebook.twig

```diff
@@ -262,6 +271,44 @@
 </div>
 
 <script>
+    //for displaying popup submit button
+    {% if is_timed and allowed_minutes > 25 %}
+        //interval is 20 percent of overall time allowed or 10 minutes, whichever is greater
+        let interval = Math.floor(((({{ allowed_minutes }}) / 5) > 10 ? ({{ allowed_minutes }}) / 5 : 10));
+        let loadTime = Date.now();
+       
+        function initializePopupTimer(){
+            sessionStorage.setItem("show_popup_submit", "yes");
+            syncWithServer(true);
+            count();
+        }
```

Comment:
This function should not call syncWithServer. It was only designed to be called from updateTime. You calling it will make the gradeable timer start up twice and run two instances. If you want to run something after syncWithServer is run you can place it in the initializeTimer in submission-page.js. Or you could assume syncWithServer gets run since it will always be there if is_timed is true.

Reply:
I just removed it, It was unnecessary in the first place.